integ/resources/cfn-kinesis-s3-firehose.yml

```diff
@@ -42,8 +42,18 @@ Resources:
     Type: AWS::Kinesis::Stream
     Properties:
       ShardCount: 5
+      StreamEncryption:
+        EncryptionType: KMS
+        KeyId: alias/aws/kinesis
   s3Bucket:
     Type: AWS::S3::Bucket
+    Properties:
+      VersioningConfiguration:
+        Status: Enabled
+      BucketEncryption:
+        ServerSideEncryptionConfiguration: 
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: AES256
```

Comment:
And the official doc link for this config: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket-serversideencryptionbydefault.html